esp/src/src-react/components/DaliCount.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { parser } from "dojo/main";
+import { DaliService } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+const logger = scopedLogger("src-react/components/DaliCount.tsx");
+
+const myDaliService = new DaliService({ baseUrl: "" });
+
+interface CountRequestProps {
+
+}
+
+export const DaliCount: React.FunctionComponent<CountRequestProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [path, setPath] = React.useState<string>("");
+
+    const onSubmit = React.useCallback(() => {
+        myDaliService.Count({ Path: path }).then(response => {
+            const data = parser.parse(response.Result);
```

Comment:
The import for parser doesn't seem to be correct. The only similar import I could find is `import * as parser from "dojox/xml/parser";` (eg, https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/src/ESPGraph.ts#L4 and a few other places). But I don't think that's what you want either, since response.Result is a number, rather than something to be parsed, as you and Kevin were discussing.

I think you'd need to define data and columns as arrays containing one item. Maybe something like:

<pre>
const data = [{
  key: "Result",
  result: response.Result
}];

const columns = [{
  key: "Result",
  name: "Result",
  fieldName: "result",
  minWidth: 100
}];
</pre>

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist